Capfile

```diff
@@ -21,10 +21,10 @@ require 'capistrano/deploy'
 # require 'capistrano/rails/assets'
 # require 'capistrano/rails/migrations'
 
-require 'capistrano/bundler'
-require 'capistrano/rails'
+#require 'capistrano/bundler'
+#require 'capistrano/rails'
 require 'dlss/capistrano'
-require 'capistrano/rvm'
+#require 'capistrano/rvm'
```

Comment:
The `Gemfile` will likely need these in the `:deployment` section -- I don't think `dlss-capistrano` requires all of them as dependencies.